cmd/texttosource/main.go

```diff
@@ -35,5 +33,9 @@ package %s
 
 const %s = %s
 `, fname, packageName, varname, bodyConstant)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to write to %s: %v\n", outname, err)
+			os.Exit(1)
+		}
```

Comment:
what is this?

Reply:
I mean, why is there an error check here, does texttosource still compile?

Reply:
I'm guessing it currently works because the compiler misses the unused `err` because it's in a loop and it thinks maybe err could be used on the next pass through the loop? Anyway, this is a good change to catch the err, print, and exit.